include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -2099,6 +2099,12 @@ FUNCTION(Free, c, free, C_CC, AlwaysAvailable,
          NO_ATTRS,
          EFFECT(Deallocating),
          UNKNOWN_MEMEFFECTS)
+FUNCTION(CoroFrameAlloc, c, swift_coroFrameAlloc, C_CC, AlwaysAvailable,
```

Comment:
This is incorrect. `swift_CoroFrameAlloc` is not defined in the C library, it is part of the Swift library.

Reply:
@compnerd Thank you for catching this and also fixing it in the `main` branch.